### PR TITLE
add warning and error messages for phase output

### PR DESCRIPTION
**Pull request by awegsche**

### CODE REVIEW

**Review by fsoubelet — APPROVED** [review]
I am ok with this

File: omc3/optics_measurements/beta_from_phase.py [hunk 1]
```diff
@@ -80,6 +80,12 @@ def n_bpm_method(meas_input, phase, plane, meas_and_mdl_tunes):
     Returns:
         `TfsDataFrame` containing betas and alfas from phase.
     """
+
+    if phases['MEAS'].count() < 3:
```
Comment (JoschD) [inline omc3/optics_measurements/beta_from_phase.py:84]: why not raise an error?

File: omc3/optics_measurements/phase.py [hunk 2]
```diff
@@ -66,6 +66,14 @@ def calculate(
                                                                              no_errors)
         dfs = free_dfs + drv_dfs
 
+    if phase_advances["MEAS"].count() < 3:
+        LOGGER.warning("Dear user, phase analysis yields less than 3 BPMs.")
```
Comment (JoschD) [inline omc3/optics_measurements/phase.py:70]: I'd write it into one single log with `\n` to separate lines. That makes it more obvious that it's one message. Also, `Dear user` might be too much

File: omc3/optics_measurements/beta_from_phase.py [hunk 3]
```diff
@@ -398,6 +404,8 @@ def three_bpm_method(meas_input, phase, plane, meas_and_mdl_tunes):
     Returns:
         `TfsDataFrame` containing betas and alfas from phase.
     """
+    if phase["MEAS"].count() < 3:
```
Comment (JoschD) [inline omc3/optics_measurements/beta_from_phase.py:407]: same as above. raise error?

**Review by JoschD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**